.github/workflows/ci.yml

```diff
@@ -10,26 +10,57 @@ on:
 jobs:
   job1:
     name: Nim Tests
-    runs-on: ubuntu-18.04
-    container: nimlang/nim:1.2.4-ubuntu-regular@sha256:02a555518a05c354ccb2627940f6138fca1090d198e5a0eb60936c03a4875c69
+    runs-on: ubuntu-latest
```

Comment:
Just a side note: I'm usually in favor of pinning versions in CI, to prevent accidental breakages when things are upgraded.

Reply:
Yes. That was one of the things I was going to get to